src/Worker/DiscoverPoCo.php

```diff
@@ -0,0 +1,304 @@
+<?php
+/**
+ * @file src/Worker/DiscoverPoCo.php
+ */
+namespace Friendica\Worker;
+
+use Friendica\Core\Cache;
+use Friendica\Core\Config;
+use Friendica\Core\Worker;
+use Friendica\Database\DBM;
+use Friendica\Model\GlobalContact;
+use Friendica\Network\Probe;
+use Friendica\Protocol\PortableContact;
+
+require_once 'include/datetime.php';
+
+class DiscoverPoco {
```

Comment:
Does it matter that all of your Worker::add calls have DiscoverPoCo but this is named DiscoverPoco?

Reply:
I second that, and suggests `DiscoverPortableContact`.

Reply:
I see another conversation between you and @annando about long names in your future!! :laughing:

Reply:
What about "DiscoverPortableContactsInTheNetworkAndTheWholeInternet"? :-)

Reply:
:rofl:

Reply:
Thing is: This class should be replaced anyway with separate classes.

Reply:
Nice deflection!